src/sentry/models/group.py

```diff
@@ -212,9 +212,12 @@ def from_event_id(self, project, event_id):
         return Group.objects.get(id=group_id)
 
     def filter_by_event_id(self, project_ids, event_id):
+        event_ids = [event_id]
+        conditions = [["group_id", "IS NOT NULL", None]]
         data = eventstore.get_events(
-            conditions=[["group_id", "IS NOT NULL", None]],
-            filter_keys={"event_id": [event_id], "project_id": project_ids},
+            filter=eventstore.Filter(
+                event_ids=event_ids, project_ids=project_ids, conditions=conditions
```

Comment:
there is no project_ids field in the Filter class constructor.

Reply:
Thanks, settled on the pluralized versions